client/components/breadcrumb/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Gridicon } from '@automattic/components';
+import styled from '@emotion/styled';
+import { Key } from 'react';
+
+const StyledUl = styled.ul`
+	display: flex;
+	align-items: center;
+	list-style-type: none;
+	margin: 0;
+`;
+
+const StyledLi = styled.li`
+	display: flex;
+	align-items: center;
+	font-size: 13px;
+	font-weight: 500;
+	--color-link: var( --studio-gray-100 );
+
+	&.first {
+		font-size: 16px;
+		font-weight: 600;
+		--color-link: var( --studio-gray-80 );
+	}
+
+	&.last {
+		--color-link: var( --studio-gray-50 );
+	}
+`;
+
+const StyledGridicon = styled( Gridicon )`
+	margin: 0 7px;
+	fill: var( --studio-gray-50 );
+`;
+
+interface Props {
+	items: { label: string; href?: string }[];
+}
+
+const Breadcrumb: React.FunctionComponent< Props > = ( { items } ) => {
+	return (
+		<StyledUl>
+			{ items.map( ( item: { href?: string; label: string }, index: Key, { length } ) => {
+				// eslint-disable-next-line no-nested-ternary
+				const className = index === 0 ? 'first' : index === length - 1 ? 'last' : '';
```

Comment:
Nitpick: Any reason to be skipping this lint rule? Is it considered "simple enough"?

Reply:
If you got it right with the first time reading it, yes I would consider it simple. I will leave this up to you!

Reply:
This one I was able to get easily on the first time, but I usually find nested ternaries tricky that's why I try to avoid them.

Reply:
Thanks for pushing on this! It made me think out of the box, that we actually don't even need JS to tell which is first and which is last.. Here is a follow up PR https://github.com/Automattic/wp-calypso/pull/57637